pynest/nest/tests/test_spatial/test_spatial_kernels.py

```diff
@@ -539,8 +539,8 @@ def z_test(self):
         if variance_num_targets == 0:
             return np.nan, 1.0
         else:
-            sd = math.sqrt(variance_num_targets)
-            z = abs((num_targets - expected_num_targets) / sd)
+            sr = math.sqrt(variance_num_targets)
```

Comment:
Should this actually be `sd`? I don't think it has anything to do with `spike_recorders`, but I am not sure.

Reply:
Argh! Thanks. Fixed!